### PR TITLE
[Enhancement] Add check compaction RPC for lake tablet

### PR DESCRIPTION
**Pull request by sduzh**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: gensrc/proto/lake_service.proto [hunk 1]
```diff
@@ -20,6 +20,8 @@ message PublishVersionRequest {
 
 message PublishVersionResponse {
     repeated int64 failed_tablets = 1;
+    // Mapping from tablet id to compaction score.
+    map<int64, double> compaction_scores = 2;
```
Comment (imay) [inline gensrc/proto/lake_service.proto:24]: Should add modifier required/optional

Comment (sduzh) [reply gensrc/proto/lake_service.proto:24]: > Should add modifier required/optional

Maps cannot be `repeated`, `optional` or `required`. 
https://developers.google.com/protocol-buffers/docs/proto#maps

**Comment by sduzh** [comment]
run starrocks_be_unittest

**Comment by sduzh** [comment]
run starrocks_clang

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by sduzh** [comment]
run starrocks_be_unittest

**Review by wyb — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Pull request merged**